modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -294,6 +294,9 @@ export type TxRequest = {
     signatureShares: SignatureShareRecord[];
     combineSigShare?: string;
     txHash?: string;
+    messageRaw: string;
+    derivationPath: string;
```

Comment:
this may need to be optional until we backfill

Reply:
Yea, it's technically optional, but for all new requests it should exist. 

Create a ticket for the backfill, but it shouldn't realistically impact anything if older models don't have it set (since they wouldn't be going through the signing flow again)